packages/waas-ethers/src/signer.ts

```diff
@@ -0,0 +1,127 @@
+import { BigNumber, ethers } from 'ethers'
+import { CommonAuthArgs, Sequence, networks } from '@0xsequence/waas'
+
+export class SequenceSigner extends ethers.Signer {
+  constructor(
+    private readonly sequence: Sequence,
```

Comment:
Maybe is it worth to have either an static method or the option to pass the sequence options directly here? so the wrapping doesn't have to be done manually